stable_baselines3/common/base_class.py

```diff
@@ -273,7 +273,7 @@ def logger(self) -> Logger:
 
     def _setup_lr_schedule(self) -> None:
         """Transform to callable if needed."""
-        self.lr_schedule = get_schedule_fn(self.learning_rate)
+        self.lr_schedule = FloatConverterSchedule(self.learning_rate)
```

Comment:
I would rather have a new `get_schedule` helper here because `FloatConverterSchedule` does more than what its name suggest

PS: you don't have to force push for every edit, you can simply push new commits to the same branch

Reply:
or maybe rename `FloatConverterSchedule` to something else (the issue is that `Schedule` type is already taken... maybe `FloatSchedule`?

Reply:
I have renamed it to FloatSchedule, as recommended, but maybe a `ScheduleWrapper` could be a better name, since it is basically a wrapper that ensures that a constant is transformed to `Schedule`,   and ensures that any callable returning float values.

I wanted to change the original logic as minimal as possible, but maybe the original function is not clean enough, since it is doing multiple things.